docs/general/rendering.md

```diff
@@ -260,15 +260,20 @@ Note that by using a string and symbol, Ruby will assume the namespace is define
 
 #### serializer
 
-PR please :)
+Specify which serializer to use if you want to use a serializer other than the default.
+
+```ruby
+@post = Post.first
+render json: @post, serializer: SpecialPostSerializer
```

Comment:
Please amend to show that you explicit serializer for a single resource can be specified by the `serializer` option, and for a collection of all the same resource by `each_serializer`.  (Unfortunately,) when rending a collection, like an index action, `serializer` refers to the collection serializer.